### PR TITLE
French translation update

### PR DESCRIPTION
**Pull request by Yvan-Masson**
Proposal of French translation update. Note that:
- I did not translated some strings because I could not find their context
- I could not find a proper translation for "Hole-filled seamless SRTM data", so I removed the "Hole-filled seamless" part of the string

### CODE REVIEW

**Comment by sfendrich** [comment]
Thank you very much for this great contribution!
 
>     * I did not translated some strings because I could not find their context

Would you mind to check my (upcoming) suggestions and comments to these open items, so we can complete the translation?

File: app/languages/fr-FR.json [hunk 1]
```diff
@@ -147,77 +147,77 @@
...
-    "LOCALE_ISOCHRONE_RANDOMIZE_COLORS": "Randomized isochrone colors",
-    "DRAGTOADD": "Drag to add a via point. Click for point info",
-    "noise": "Noise index (for Germany only)",
-    "height": "Height",
+    "LOCALE_MAP_SETTINGS": "Réglages de la carte",
+    "LOCALE_ISOCHRONE_RANDOMIZE_COLORS": "Couleurs isochrones aléatoires",
+    "DRAGTOADD": "Faire glisser pour ajouter un point de passage. Cliquer pour un point d'information",
+    "noise": "Index de bruit (Allemagne uniquement)",
+    "height": "Hauteur",
     "LOCALE_DISTANCE_MARKERS": "Distance markers (km)",
```
Comment (sfendrich) [inline app/languages/fr-FR.json]: Marqueurs *ou* marquages de distance?

File: app/languages/fr-FR.json [hunk 2]
```diff
@@ -147,77 +147,77 @@
...
+    "CONTROLLEDBORDERS": "Frontières avec contrôles",
+    "BORDERS": "Toutes les frontières",
+    "COUNTRYBORDERS": "Frontières de pays",
+    "SEARCHAVOIDCOUNTRIES": "Rechercher des pays à éviter",
     "TOTALPOP": "Population",
-    "COPYYX": "Copy Lat,Lng",
-    "COPYXY": "Copy Lng,Lat",
+    "COPYYX": "Copier Lat,Lng",
+    "COPYXY": "Copier Lng,Lat",
     "SHOWLANDMARKS": "Include landmarks in routing instructions",
```
Comment (sfendrich) [inline app/languages/fr-FR.json]: Peut-être "amer" (landmark)?

File: app/languages/fr-FR.json [hunk 3]
```diff
@@ -147,77 +147,77 @@
...
-    "SEARCH": "Search",
-    "OPTIONAL_PARAMS": "Optional parameters:",
-    "CALC_CONSUMPTION": "Calculate consumption",
-    "_RESET": "Reset",
+    "FUEL_COSTS": "Coût du carburant",
+    "SEARCH": "Recherche",
+    "OPTIONAL_PARAMS": "Paramètres optionnels :",
+    "CALC_CONSUMPTION": "Calculer la consommation",
+    "_RESET": "Réinitialiser",
     "ON_ROUTE_CHANGE": "on route change",
```
Comment (sfendrich) [inline app/languages/fr-FR.json]: @MichaelsJP Is this also related to openfuelservice?

File: app/languages/fr-FR.json [hunk 4]
```diff
@@ -147,77 +147,77 @@
...
+    "CAR_BRAND": "Marque",
+    "DRIVING_SPEED": "Vitesse de conduite",
+    "VEHICLE_CATEGORY": "Type du véhicule",
+    "TANK_SIZE": "Capacité du réservoir",
     "SPECIFIC_FUEL_CONSUMPTION": "Specific fuel consumption",
-    "CAR_MODEL": "Car model",
-    "CAR_YEAR": "Construction year",
+    "CAR_MODEL": "Modèle",
+    "CAR_YEAR": "Année de fabrication",
     "CAR_TYPE": "Special type",
```
Comment (sfendrich) [inline app/languages/fr-FR.json]: @MichaelsJP What is this about?

File: app/languages/fr-FR.json [hunk 5]
```diff
@@ -147,77 +147,77 @@
...
-    "DRIVING_SPEED": "Driving speed",
-    "VEHICLE_CATEGORY": "Vehicle category",
-    "TANK_SIZE": "Tank size",
+    "FUEL_CONSUMPTION": "Consommation de carburant (expérimental)",
+    "FUEL_TYPE": "Type de carburant",
+    "CAR_BRAND": "Marque",
+    "DRIVING_SPEED": "Vitesse de conduite",
+    "VEHICLE_CATEGORY": "Type du véhicule",
+    "TANK_SIZE": "Capacité du réservoir",
     "SPECIFIC_FUEL_CONSUMPTION": "Specific fuel consumption",
```
Comment (sfendrich) [inline app/languages/fr-FR.json]: @MichaelsJP What is this about?

File: app/languages/fr-FR.json [hunk 6]
```diff
@@ -147,77 +147,77 @@
...
-    "INFO_SUPPORT_TEXT": "Openrouteservice offers free services and relies on donations for a majority of its funding. If you would like to support the further development of features and server infrastructure donations are very welcome.",
-    "LOCALE_HEIGHTGRAPH": "Elevation profile",
-    "DEVELOPER_SETTINGS": "Developer Settings",
+    "CUSTOM_MARKER": "Ajouter un marqueur personnalisé sur la carte",
+    "ELEVATION": "Altitude",
+    "INFO_SUPPORT": "Merci d'aider openrouteservice.org",
+    "INFO_SUPPORT_TEXT": "Openrouteservice offre des services gratuits et est financé en majorité par des dons. Si vous souhaitez aider au développement de nouvelles fonctionnalités et à la maintencance de l'infrastructure serveur, les dons sont particulièrements appréciés.",
+    "LOCALE_HEIGHTGRAPH": "Profil altimétrique",
+    "DEVELOPER_SETTINGS": "Paramètres développeurs",
     "INCLUDE_INSTRUCTIONS": "Include Instructions",
```
Comment (sfendrich) [inline app/languages/fr-FR.json]: "Inclure déscription de l'itinéraire"?

File: app/languages/fr-FR.json [hunk 7]
```diff
@@ -147,77 +147,77 @@
...
+    "noise": "Index de bruit (Allemagne uniquement)",
+    "height": "Hauteur",
     "LOCALE_DISTANCE_MARKERS": "Distance markers (km)",
-    "LOCALE_NO_TOLLWAY": "Tollfree roads",
-    "LOCALE_TOLLWAY": "Tollroads ",
-    "avgspeed": "Average speed",
+    "LOCALE_NO_TOLLWAY": "Sections gratuites",
+    "LOCALE_TOLLWAY": "Sections à péage ",
+    "avgspeed": "Vitesse moyenne",
     "traildifficulty": "Trail difficulty",
```
Comment (sfendrich) [inline app/languages/fr-FR.json]: "Difficulté de sentier" dans le contexte de rendonnées.

Comment (Yvan-Masson) [reply app/languages/fr-FR.json]: I think "Difficulté de sentier" is OK.

File: app/languages/fr-FR.json (hunk 1 above)
Comment (Yvan-Masson) [reply app/languages/fr-FR.json]: I would say "Marqueurs de distance (km)".

File: app/languages/fr-FR.json (hunk 2 above)
Comment (Yvan-Masson) [reply app/languages/fr-FR.json]: Sorry, I don't understand what you said. I just saw that this is an option for pedestrian routing: could you explain what is it used for?

File: app/languages/fr-FR.json (hunk 6 above)
Comment (Yvan-Masson) [reply app/languages/fr-FR.json]: OK, I just found where this string is used (it is associated with a checkbox when you want to export the route). I would rather suggest "Inclure les instructions de routage".

File: app/languages/fr-FR.json (hunk 5 above)
Comment (TheGreatRefrigerator) [reply app/languages/fr-FR.json]: grafik

File: app/languages/fr-FR.json (hunk 3 above)
Comment (TheGreatRefrigerator) [reply app/languages/fr-FR.json]: yes

File: app/languages/fr-FR.json (hunk 4 above)
Comment (TheGreatRefrigerator) [reply app/languages/fr-FR.json]: grafik
Upper select fields

File: app/languages/fr-FR.json (hunk 2 above)
Comment (TheGreatRefrigerator) [reply app/languages/fr-FR.json]: Currently this does nothing because the functionality somehow is broken.
For pedestrian routes this could be chosen to give routing instructions according to landmarks in addition to streetnames (like turn left after big church on the right)

Comment (Yvan-Masson) [reply app/languages/fr-FR.json]: Ok thanks, so this could be translated to "Ajouter des repères dans les instructions de routage".

File: app/languages/fr-FR.json (hunk 3 above)
Comment (Yvan-Masson) [reply app/languages/fr-FR.json]: Could someone explain what this option means so that I can propose a translation?

File: app/languages/fr-FR.json (hunk 5 above)
Comment (sfendrich) [reply app/languages/fr-FR.json]: Sorry, this does not really answer my question. In order to give a sensible translation, we need to know what "SPECIFIC_FUEL_CONSUMPTION" means.

File: app/languages/fr-FR.json (hunk 4 above)
Comment (Yvan-Masson) [reply app/languages/fr-FR.json]: Thanks for the explanation. I believe in French it is "Série"

File: app/languages/fr-FR.json (hunk 3 above)
Comment (TheGreatRefrigerator) [reply app/languages/fr-FR.json]: this is for the checkbox below the calculate (fuel) consumption button in the experimental fuel service settings
When checked the fuel consumption is calulated on each route calculation instead of only when the button is pressed

File: app/languages/fr-FR.json (hunk 4 above)
Comment (sfendrich) [reply app/languages/fr-FR.json]: Sorry, this does not really answer my question. In order to give a sensible translation, we need to know what the parameter means.

File: app/languages/fr-FR.json (hunk 3 above)
Comment (sfendrich) [reply app/languages/fr-FR.json]: And what does it mean?

File: app/languages/fr-FR.json (hunk 5 above)
Comment (Yvan-Masson) [reply app/languages/fr-FR.json]: Ok, so maybe "Consommation moyenne" is the best.

Comment (TheGreatRefrigerator) [reply app/languages/fr-FR.json]: this can be set instead for overwriting the default vehicle category consumption so this consumption specific consumption will be used instead

File: app/languages/fr-FR.json (hunk 4 above)
Comment (Yvan-Masson) [reply app/languages/fr-FR.json]: If I understand it well, this option allows specifying the special type of the car (mainly to know which motor it has I believe).

Comment (sfendrich) [reply app/languages/fr-FR.json]: Ok. I believe "Série" is good. Maybe also in english "series" would be the better term.

File: app/languages/fr-FR.json (hunk 5 above)
Comment (sfendrich) [reply app/languages/fr-FR.json]: I agree,  "Consommation moyenne" sounds good.

File: app/languages/fr-FR.json (hunk 3 above)
Comment (sfendrich) [reply app/languages/fr-FR.json]: ON_ROUTE_CHANGE concerns a checkbox that makes the app recalculate the fuel-consumption when changing the way points of the route on the map.

Comment (Yvan-Masson) [reply app/languages/fr-FR.json]: Thanks. I would say "Mettre à jour automatiquement", which literally means "Update automatically".

**Comment by sfendrich** [comment]
@Yvan-Masson Thanks again for your work. Could you update your PR according to our discussions? Afterwards we'll be ready to merge.
